### PR TITLE
add teamID in both getall and getby id , also add list of scorecardna…

### PR DESCRIPTION
**Pull request by MaiyoiaGamal**
add teamid in get all and get by id 
also add scorecard names list in getall endpiont

### CODE REVIEW

File: app/crud/microservice.py [hunk 1]
```diff
@@ -31,13 +32,23 @@ def getAllServicesWithTeamName(self) -> list[MicroserviceInDBBase]:
         
         for microservice in microservices:
             team = self.db_session.query(Team).filter(Team.id == microservice.teamId).first()
-            
+            scorecard_names = (
```
Comment (Passant-Abdelgalil) [inline app/crud/microservice.py:35]: why is this needed ?

Comment (MaiyoiaGamal) [reply app/crud/microservice.py:35]: ammar need teamid to use it

**Review by Passant-Abdelgalil — APPROVED** [review]
